integration-cli/docker_api_inspect_test.go

```diff
@@ -159,14 +159,19 @@ func (s *DockerAPISuite) TestInspectAPIBridgeNetworkSettings120(c *testing.T) {
 	assert.Assert(c, len(settings.IPAddress) != 0)
 }
 
+// Inspect for API v1.21 and up; see
```

Comment:
Does this comment make any sense at all now that we don’t do anything with the api version?

Reply:
Honestly, I didn't want to go through the "disable integration-cli check -> merge -> re-enable" cycle to rename the test (removing 121), and thought I'd leave some crumble-patch to why it was named like this 😂

Reply:
Oh, was on my phone, didn't see the test name

Reply:
No worries; it's a good question, and perhaps it's "too much info". I also anticipated that _perhaps_ we'd have version-dependent changes again in future (but likely we should rewrite / move this test to `integration` then).